src/layouts/components/Navigation.astro

```diff
@@ -38,6 +38,14 @@ import { navigationLinks } from "@data/links";
   .nav-logo {
     width: 4rem;
     height: 4rem;
+
+    :global(svg *) {
+      animation-play-state: paused;
+    }
+
+    &:hover :global(svg *) {
+      animation-play-state: running;
+    }
```

Comment:
We should probably do ourselves a favour going forward to _always_ document the reasoning for a `:global` style.

Reply:
It's mostly due to some quirks with how Astro does the CSS scoping. At the end of the day it just needs to boil down to `.nav-logo svg *` to function. If we find a better way to achieve that I'd be stoked to use that instead.

Happy to add a comment about it into the file itself. c: